src/map/Pins.js

```diff
@@ -80,17 +80,18 @@
         var context = this;
         var gPinEnter = this.pinsPaths.enter().append("g").attr("class","pin");
         gPinEnter
-            .append("path")
-            .attr("class", "data")
-            .call(this._selection.enter.bind(this._selection))
             .on("click", function (d) {
                 context.click(context.rowToObj(d[2].origRow), "geohash", context._selection.selected(this));
             })
             .on('mouseover', function (d) {
                 if (!this.isIE) {
                     this.parentNode.appendChild(this);
                 }
-            })
+            });
+        gPinEnter
+            .append("path")
+            .attr("class", "data")
+            .call(this._selection.enter.bind(this._selection))
```

Comment:
*this._selection* is initialized as an empty instance of **SimpleSelection** so it will never be null